mmpose/models/keypoint_heads/temporal_regression_head.py

```diff
@@ -41,9 +45,33 @@ def __init__(self,
         self.conv = build_conv_layer(
             dict(type='Conv1d'), in_channels, num_joints * 3, 1)
 
+        if self.max_norm is not None:
+            # Apply weight norm clip to conv layers
+            weight_clip = WeightNormClipRegister(self.max_norm)
+            for module in self.modules():
+                if isinstance(module, nn.modules.conv._ConvNd):
+                    weight_clip.register(module)
+
+    def _transform_inputs(self, x):
+        """Transform inputs for decoder.
+
+        Args:
+            inputs (tuple/list of Tensor | Tensor): multi-level img features.
+
+        Returns:
+            Tensor: The transformed inputs
+        """
+        if not isinstance(x, (list, tuple)):
+            return x
+
+        assert len(x) > 0
+        return x[-1]
```

Comment:
So it simply drops all inputs other than the last one?

Reply:
Yes for now. The backbone outputs multi-level features (1D feature pyramid?) but only the top-level is used by the head.

Reply:
Okay, add a line of comment like

` # return the top-level feature of the 1D feature pyramid`

We would like to give explanations to every magic number